lib/improver/cli/apply_emos_coefficients.py

```diff
@@ -75,8 +78,8 @@ def main(argv=None):
              'probabilities or percentiles.')
     parser.add_argument(
```

Comment:
I guess we are relying on the filelist generation to provide these filenames in the right order. Although if one is missing the one provided will be assumed to be a forecast_filepath. Should we put something in to check that the coefficient cube is a coefficient cube and that the forecast cube is a forecast cube? 
Although we do depend on the order of the filenames in other CLIs. The alternative is to separate out the cubes in the CLI with just one input filenames argument to the CLI. Any thoughts?